brickstrap.sh

```diff
@@ -57,6 +58,18 @@ Options
                This option is required and should occur exactly once.
                Values are either a path to the project directory or the name of
                an example project shipped with brickstrap by default.
+-Q <qemu>      Specify which QEMU binary to use for emulation of foreign
+               instruction sets; this may be used to override auto-detected
+               QEMU.
```

Comment:
Could we just say "Optionally override the qemu binary used to emulate a foreign architecture." I think this makes it a bit more clear that this is an optional parameter and is bit more succinct.